web/package.json

```
@@ -3,8 +3,8 @@
     "private": true,
     "scripts": {
         "test": "eslint . && tsc --noEmit && jest --coverage",
-        "build": "gulp prod",
-        "start": "gulp",
+        "build": "rm -rf ../mitmproxy/tools/web/static && vite build",
```

AFAIK `rm` doesn’t exist in classic Command Prompt (CMD) on Windows. I'm a proud Arch Linux user myself, but we should take other operating systems into account.

A classic way to solve this is to install a cross-platform utility package such as [rimraf](https://www.npmjs.com/package/rimraf):

```
npm i --save-dev rimraf
```

```suggestion
        "build": "rimraf ../mitmproxy/tools/web/static && vite build",
```